src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -268,11 +268,6 @@ class HuiGaugeCard extends LitElement implements LovelaceCard {
         box-sizing: border-box;
       }
 
-      ha-card:focus {
-        outline: none;
-        background: var(--divider-color);
```

Comment:
I agree this is the wrong color to use here, but we should still have a indicator the card is focussed. I suggest we use `background-color: var(--mdc-ripple-color, #000);` with an opacity of 0.12 as an `before`, the same as the button card has.

Reply:
Are you suggesting theres something special about the gauge card that it needs a focus indicator? Other display type cards do not have this. Entity card, sensor card, glance card, graph card, weather card, etc.  don't have a focus indicator. Should they all be consistent, are you asking to add this to all those cards? 

And why opacity of 0.12, isn't that essentially close to fully transparent? (The thing I am trying to remove here?) Though I may not be understanding by what you mean "as an `before`". 

Entities card has focus on individual row icons only, not the text. Tile card has focus on icon and text separately as two elements. 

I'd need some guidance where to go from here, if you don't want to accept this change.

Reply:
I agree with @karwosts. The current behavior seems inconsistent. No other card can be focused visually, only elements on them.

Which makes me actually think we need the `outline: none` part since otherwise browser could potentially draw a focus outline (I just remembered this PR of mine https://github.com/home-assistant/frontend/pull/14535) and should only remove the background color statement.

Reply:
The gauge is an interactive element? You can click it and it will open the more info?

Reply:
Every element that is interactive and is focusable should have a focus indicator. That other cards are wrong does not make it right to remove it from this card.

Reply:
For accessibility, @bramkragten is definitely correct that interactive elements need a visual indicator that they are focused.  And that indicator needs to also have sufficient color contrast.

That said, this is a small part of a much larger problem with regard to more-info dialogs (and also user configurable tap actions).  Many cards or parts of cards are not even focusable via `tabindex`, let alone have a visual indicator.  They also do not have proper interactive ARIA roles.

I've been avoiding tackling this issue because it's quite complicated and cards are not consistent as you're finding out here.  The workaround for accessibility is just to learn that these things are clickable and not rely on tab to navigate them.

Reply:
Bram if you don't want this current change, I have to admit I still don't understand your suggestion:

>  I suggest we use background-color: var(--mdc-ripple-color, #000); with an opacity of 0.12 as an before, the same as the button card has.

I looked at the button card it doesn't have anything like a background color, 

Either maybe you want to elaborate a bit, or if not I can just close the pr, but I would be sad leaving the gauge the way it currently is, that looks pretty terrible in the gif in the initial post.

Reply:
The button card uses the `mwc-ripple`